src/open-response/components/report-item/init-message-info.tsx

```diff
@@ -0,0 +1,24 @@
+import * as React from "react";
+// import { useRef } from "react";
+import { IReportItemInitInteractive } from "@concord-consortium/lara-interactive-api";
+
+interface Props {
+  initMessage: IReportItemInitInteractive;
+}
+
+export const InitMessageInfoComponent: React.FC<Props> = (props) => {
+  // const initMessage = useRef(props.initMessage);
+  // const {users, interactiveItemId, view} = initMessage.current;
+  // const userIds = Object.keys(users);
+  // const numAnswersAtInit = userIds.reduce<number>((acc, studentId) => {
+  //   return users[studentId].hasAnswer ? acc + 1 : acc;
+  // }, 0);
+  // const userAnswers = useRef<Record<string, any>>({});
+  // const numCurrentAnswers = Object.keys(userAnswers).length;
+
+  return (
+    // TODO: Figure out what, if anything, has to be shown here.
```

Comment:
I modeled the new open response report item on the example interactive for attachments. Although the open response report item seems to be necessary for making the attachment information accessible on the portal-report side, it doesn't seem like it needs to render anything to be shown in the portal-report. Does that make sense, though?

Reply:
Yes - I think this whole file can be removed along with the two summary componenents.